### ISSUE TITLE
Getting HTTP 400 - "UNMAPPED_ERROR" when updating a billing plan to ACTIVE.

### ISSUE THREAD

**Issue by whYME953** [paypal/PayPal-REST-API-issues#237]
### General information

* Environment: Sandbox
* `PayPal-Debug-ID` values: e1613b9241848
* Language, language version, and OS: PHP SDK 1.13.0

### Issue description

I'm getting an HTTP response code 400 - "UNMAPPED_ERROR" when trying to update a billing plan to ACTIVE.

>PayPal\Core\PayPalHttpConnection : DEBUG: Request Data		: [{"op":"replace","path":"/","value":"{\"state\":\"ACTIVE\"}"}]

> PayPal\Core\PayPalHttpConnection : ERROR: Got Http response code 400 when accessing https://api.sandbox.paypal.com/v1/payments/billing-plans/P-**************************. {"name":"UNMAPPED_ERROR","details":[{"field":"unmapped error","issue":"Internal error occured"}],"message":"Unmapped error","information_link":"https://developer.paypal.com/docs/api/payments.billing-plans#errors","debug_id":"b332ccff3ecd1"}

The debug ID above was using the following code:
`$patch = new Patch();
$patch->setOp('replace') ->setPath('/') ->setValue('{"state":"ACTIVE"}');
$patchRequest = new PatchRequest();
$patchRequest->addPatch($patch);

$plan = Plan::get($id, $apiContext);
$plan->update($patchRequest, $apiContext);`

**Comment by trainerbill** [comment]
You are setting the value to a string and it needs to be an object.  Take a look at how it works in the sample.  https://github.com/paypal/PayPal-PHP-SDK/blob/master/sample/billing/UpdatePlan.php#L24

**Comment by whYME953** [comment]
Thanks!

I could've _sworn_ I had tried that yesterday, as well as passing an array instead and neither of those worked either. Oh well.

Regardless, it's working now by passing an array, so thank you.


>it needs to be an object.

In the docs it's listed as mixed value. 
http://paypal.github.io/PayPal-PHP-SDK/docs/class-PayPal.Api.Patch.html
I glanced through the code yesterday and thought that a string was a valid value as well but I must've mixed it up with something else
